WooCommerce/Classes/ViewRelated/Dashboard/JetpackInstall/JetpackInstallIntroView.swift

```diff
@@ -0,0 +1,116 @@
+import SwiftUI
+
+/// Hosting controller wrapper for `JetpackInstallIntroView`
+///
+final class JetpackInstallHostingController: UIHostingController<JetpackInstallIntroView> {
+    init(siteURL: String) {
+        super.init(rootView: JetpackInstallIntroView(siteURL: siteURL))
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setDismissAction(_ dismissAction: @escaping () -> Void) {
+        rootView.dismissAction = dismissAction
+    }
+}
+
+/// Displays the intro view for the Jetpack install flow.
+///
+struct JetpackInstallIntroView: View {
+    // Closure invoked when Close button is tapped
+    var dismissAction: () -> Void = {}
+
+    let siteURL: String
```

Comment:
nit: maybe this can be in the initializer so that the property isn't accessible externally?